### PR TITLE
1.19.4 + migrate to Fabric

### PR DESCRIPTION
**Pull request by embeddedt**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,10 +8,10 @@ maven_group = xyz.xdmatthewbx
 archives_base_name = chatlog
 
 # Dependencies
-cloth_config_version = 9.0.94
-mod_menu_version = 5.0.2
+cloth_config_version = 10.0.95
+mod_menu_version = 6.1.0-rc.4
```
Comment (Nolij) [inline gradle.properties:12]: Is there a working release version available?

File: src/main/java/xyz/xdmatthewbx/chatlog/ChatLogConfig.java [hunk 2]
```diff
@@ -261,7 +261,7 @@ public ChatLogConfig() {
 			var displayFactor = (bounds.displayFactor() * bounds.step());
 			return Collections.singletonList(ConfigEntryBuilder.create()
 				.startIntSlider(MutableText.create(
-					new TranslatableComponent(i13n)),
+					new TranslatableComponent(i13n, null, new Object[0])),
```
Comment (Nolij) [inline src/main/java/xyz/xdmatthewbx/chatlog/ChatLogConfig.java:264]: I'm sure this wasn't done without reason, but this looks very strange.

File: gradle.properties (hunk 1 above)
Comment (embeddedt) [reply gradle.properties:12]: All 1.19.4 versions are RCs.

File: src/main/java/xyz/xdmatthewbx/chatlog/ChatLogConfig.java (hunk 2 above)
Comment (embeddedt) [reply src/main/java/xyz/xdmatthewbx/chatlog/ChatLogConfig.java]: Fixed (constant was probably unmapped during snapshots, which is when I wrote this).

**Review by Nolij — CHANGES_REQUESTED** [review]
Does this work in a Quilt environment? I want to ensure Quilt support is maintained.

If it does, the README should be updated to reflect the new default modloader. I can prepare this part.

**Review by Nolij — APPROVED** [review]

**Pull request merged**